### PR TITLE
Update version of libraryDependencies

### PR DESCRIPTION
**Pull request by nobusugi246**
I appreciate your efforts on Asciidoctor Plugin.

I made a little update for libraryDependencies. Please consider to accept this.

### CODE REVIEW

**Comment by nobusugi246** [comment]
update for GitBucket ver.3.11

**Comment by lefou** [comment]
Hello @nobusugi246 !

Thank you for your pull request! (Unfortunately, GitHub didn't sent me any notification, so I saw it just now. I'm sorry!) I'd like to accept your PR, but have some annotations which I'll add inline.

Regards,
Tobias

**Comment by lefou** [comment]
This would also fix #7

File: README.adoc [hunk 1]
```diff
@@ -1,26 +1,33 @@
 = gitbucket-asciidoctor-plugin
-:plugin-version: 0.1.0
+:plugin-version: 0.1.2
 
-image:https://travis-ci.org/lefou/gitbucket-asciidoctor-plugin.svg["Build Status", link="https://travis-ci.org/lefou/gitbucket-asciidoctor-plugin"]
+//image:https://travis-ci.org/lefou/gitbucket-asciidoctor-plugin.svg["Build Status", link="https://travis-ci.org/lefou/gitbucket-asciidoctor-plugin"]
```
Comment (lefou) [inline README.adoc:4]: Could you please re-add the image.

File: README.adoc [hunk 2]
```diff
@@ -1,26 +1,33 @@
...
 
-A https://github.com/takezoe/gitbucket[GitBucket] plug-in that provides AsciiDoc rendering capabilities.
+A https://github.com/gitbucket/gitbucket[GitBucket] plug-in that provides AsciiDoc rendering capabilities.
 All `.adoc` (and also the deprecated `.asciidoc` and `.ad` suffix) files will be automatically rendered as AsciiDoc, supporting most features of the http://asciidoctor.org/[Asciidoctor project], including rendering of a table of contents.
 
 == Download
 
-You can download a precompiled version from the https://github.com/lefou/gitbucket-asciidoctor-plugin/releases[Release page].
+You can download a precompiled version from the https://github.com/nobusugi246/gitbucket-asciidoctor-plugin/releases[Release page].
```
Comment (lefou) [inline README.adoc:11]: I would like to have the released artifacts at the same place as the code. Could you please remove this change?

File: README.adoc [hunk 3]
```diff
@@ -28,6 +35,14 @@ This plugin is published under the Apache License, version 2.0.
 
 == ChangeLog
 
+=== gitbucket-asciidoctor-plugin 0.1.2 - 2016-01-30
```
Comment (lefou) [inline README.adoc:38]: I'm ok with a prefilled Changelog, but the release date isn't correct. At least, not for this repo, as I obviously have never release these versions.

**Comment by lefou** [comment]
I manually merged your changes. Thanks again.

**Comment by nobusugi246** [comment]
Thank you for your acceptance and sorry for late reply. It's hard to know comments timely...

**Pull request closed without merging**